src/Stryker.Core/Stryker.Core/MutantFilters/DiffMutantFilter.cs

```diff
@@ -203,6 +210,17 @@ private IEnumerable<Mutant> SetMutantStatusForFileChanged(IEnumerable<Mutant> mu
             return mutants;
         }
 
+        private IEnumerable<Mutant> SetMutantStatusForNonCSharpFileChanged(IEnumerable<Mutant> mutants)
+        {
+            foreach (var mutant in mutants)
+            {
+                mutant.ResultStatus = MutantStatus.NotRun;
+                mutant.ResultStatusReason = "A file with a different extension than .cs was changed. Cannot determine influence of file change on this mutant";
```

Comment:
This reason is a bit long. Since it will be shown to the user in the console I would prefer to keep it short.

```suggestion
                mutant.ResultStatusReason = "Non-code test project files were changed.";
```